packages/reducers/src/makeEnhancer.js

```diff
@@ -8,6 +23,29 @@ import composeReducers from './composeReducers';
 
 export const storeInterface = makeStoreInterface('reducers');
 
+const cleanUpReducer = (state, action) => {
+	if (action.type !== '@redux-tools/REDUCERS_EJECTED') return state;
+
+	const namespaceGroup = action.meta.feature || DEFAULT_FEATURE;
```

Comment:
this will fail if `meta` is not in action

Reply:
use `feature`, not `namespaceGroup`